FWCore/Framework/interface/EventSetupRecordImpl.h

```diff
@@ -150,6 +150,9 @@ namespace edm {
       void invalidateResolvers();
       void resetIfTransientInResolvers();
 
+      ComponentDescription const* componentDescription(ESResolverIndex iResolverIndex) const;
+      unsigned int transitionID(ESResolverIndex iResolverIndex) const;
+
```

Comment:
Should there be one function which returns both (rather than 2 functions)?

Reply:
Both these functions will be deleted in the latest version when I push it.